test/saithrift/tests/saiacl.py

```diff
@@ -1134,3 +1134,503 @@ def runTest(self):
+            self.client.sai_thrift_set_lag_attribute(lag_id1, attr)
+            # cleanup ACL
+            self.client.sai_thrift_remove_acl_entry(acl_entry_id)
+            self.client.sai_thrift_remove_acl_table(acl_table_id)
+            sai_thrift_remove_route(self.client, vr_id, addr_family, ip_addr3_subnet, ip_mask1, nhop1)
+            self.client.sai_thrift_remove_next_hop(nhop1)
+            sai_thrift_remove_neighbor(self.client, addr_family, rif_id2, ip_addr2, dmac1)
+            sai_thrift_remove_lag_member(self.client, lag_member_id1)
+            self.client.sai_thrift_remove_lag(lag_id1)
+            self.client.sai_thrift_remove_router_interface(rif_id1)
+            self.client.sai_thrift_remove_router_interface(rif_id2)
+            self.client.sai_thrift_remove_virtual_router(vr_id)
+            self.client.sai_thrift_remove_vlan_member(vlan_member1)
+            self.client.sai_thrift_remove_vlan(vlan_oid)
+
+
+@group('acl')
+class L3AclTableGroupTestI(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        """
+        Description:
+        Create route objects as in "L3AclTableTest_I". Create two ACL tables
+        with stage ingress, and one with destination IPv4 and other with
+        source IPv4. Create ACL entry in both the tables with source IP address
+        action drop and destination IP address action drop. Associate the two
+        tables to an ACL group with type "parallel" and bind it to the LAG.
+        Send traffic matching the destination IP and source IP and verify output
+
+        Also included L3AclTableGroupTest_III: Remove the group binding and verify
+        that traffic is now forwarded for the LAG
+
+        Steps:
+          1. Create VLAN 100 and associate untagged port 2 as the member port of VLAN 100.
+          2. Set port attribute value of "Port VLAN ID 100" for the port 2.
+          3. Create Virtual router V1 and enable V4.
+          4. Create LAG "Id1"
+          5. Remove the port 1 from default VLAN and add the port 1 as member of LAG Id1.
+          6. Create two virtual router interfaces and set the interface type as "LAG" and "VLAN" for port 1 and 2 respectively.
+          7. Create IPv4 neighbor entry (192.168.0.1) with MAC1 and associate with "RIF Id1".
+          8. Create next hop to reach the neighbor entry.
+          9. Create route entry with /24 mask (i.e 11.11.11.0/24) through NHop1
+          10. Send IPv4 packet from port 1 (LAG member) with destination IP 11.11.11.1
+
+        Part1:
+          11. Create ACL table group "acl_table_group_id"
+          12. Create two ACL table to block the traffic based on Dst IP address (11.11.11.1) and Src IP address (10.10.10.1).
+          13. Associate the ACL table with the group members.
+          14. Bind the ACL table and entry with LAG Id1 with attribute SAI_LAG_ATTR_INGRESS_ACL
+          15. Send the IP packet again through port 1 and ensure the packet is dropped.
```

Comment:
May be in next iteration, we can consider sending two different packets, one matching source IP acl entry and another one matching destination IP acl entry and ensure both are dropped. In this test, since the same packet match both the entries, the parallel action part is not really verified.